### ISSUE TITLE
Library Documentation

### ISSUE THREAD

**Issue by Southclaws** [pawn-lang/YSI-Includes#72]
Thread to discuss expanding the documentation.

I'll be up for documenting modules to the best of my ability so drop a message here asking me (or anyone else who's willing) to document how a YSI module works and I'll let you know if I can do it or not and when I expect to write it.

**Comment by Crayder** [comment]
Current, basic layout of the wiki:

```
home.md
    y_lib1
        lib1_func1
        lib1_func2
        ...
    y_lib2
        lib2_func2
        lib2_func1
        ...
    ...
y_lib1.md
    lib1_func1 section
        pawn doc description
    lib1_func2 section
        pawn doc description
    ...
y_lib2.md
    lib2_func1 section
        pawn doc description
    lib2_func2 section
        pawn doc description
    ...
...
```

Each library header in `home.md` is linked to it's page.
Each function listed in the `home.md` is linked to it's library's markdown page and it's index on that page.

Each function in the librarys' page has all of the features that standard PAWN Documentation (XML descriptions (standardized by Compuphase, they even have a program that views it all like a webpage)) includes.

**Comment by Crayder** [comment]
See https://github.com/JohnyMac/PawnDocToMarkdown to see the PAWN doc exporter me and JohnyMac made (mostly him, my idea, layout, and plans)

**Comment by Ahmad45123** [comment]
Oh, hey...  So, @Crayder I have added the ability to get files out of folder but didn't have time to add a proper dialog or test it, sorry :P, You will have to add the paths manually in the textbox, And use a _|_ as a separator...

**Comment by Y-Less** [comment]
Yes, sorry.  I know that the documentation was never up to scratch - sorry about that!  There are some bits floating about, notably here:

http://wiki.sa-mp.com/wroot/index.php?title=YSI:Main_Page&oldid=26153

and here:

http://ysi.wikia.com/wiki/YSI_Wiki

**Comment by Ahmad45123** [comment]
I've created a y_xml tutorial ages ago using some help from Y_Less, Problem that it was made in MS Word and not markdown so am gonna just upload it here and feel free to use it if its needed.

https://www.dropbox.com/s/cd4d2tv9m553g1y/XML%20Tutorial.rar?dl=0
_For some reason, Github doesn't allow to upload ZIP files anymore.._
